vmdb/app/models/server_role.rb

```diff
@@ -52,23 +52,23 @@ def self.to_role(server_role)
   end
 
   def self.all_names
-    self.find(:all, :order => :name, :select => "id, name").collect { |r| r.name }
+    self.order(:name).pluck(:name)
```

Comment:
I remember a case where we had to .select the "id" in the past even if we didn't use it because one of the callbacks (like after_find) needed it.  I can't remember if it was this class or not, but something to keep in mind to check when we remove select values.